contracts/src/v0.8/vrf/dev/VRFV2PlusWrapperConsumerBase.sol

```diff
@@ -119,4 +119,16 @@ abstract contract VRFV2PlusWrapperConsumerBase {
     require(msg.sender == address(VRF_V2_PLUS_WRAPPER), "only VRF V2 Plus wrapper can fulfill");
     fulfillRandomWords(_requestId, _randomWords);
   }
+
+  /// @notice getWrapper returns the Wrapper object used for accessing VRF service using direct billing
+  function getWrapper() external view returns (IVRFV2PlusWrapper) {
+    return VRF_V2_PLUS_WRAPPER;
```

Comment:
Is there a reason this member variable isn't a public immutable?

Reply:
Changed VRF_V2_PLUS_WRAPPER in VRFV2PlusWrapperConsumerBase to be public immutable as discussed